app/src/lib/git/diff.ts

```diff
@@ -30,13 +32,13 @@ import { DiffParser } from '../diff-parser'
  * This is a hard limit on how big a buffer can be and still be converted into
  * a string.
  */
-const MaxDiffBufferSize = 268435441
+const MaxDiffBufferSize = 268435441 //~268MB
```

Comment:
This number represents the limit where we no longer show the diff even if the user insists. 

I tried to render a diff for a file that was ~70MB (it showed the initial screen and then clicked on the show diff button) and it cut off after some point (image)
<img width="1436" alt="screen shot 2018-03-06 at 12 16 50 am" src="https://user-images.githubusercontent.com/1834387/37005423-4a4adac0-20d4-11e8-9c9f-4199a874b515.png">

Maybe we need to bring MaxDiffBufferSize to something <=70MB. It rendered a 62MB file fine.